File: py/powersupply.py
from komponen import Komponen

class PowerSupply(Komponen):
    def __init__(self, merk: str, nama: str, wattage: int):
        super().__init__(merk, nama)
        self.wattage = wattage

    def get_wattage(self):
        return self.wattage

File: java/PowerSupply.java
public class PowerSupply extends Komponen {
    private int wattage;

    public PowerSupply(String merk, String nama, int wattage) {
        super(merk, nama);
        this.wattage = wattage;
    }

    public int getWattage() {
        return wattage;
    }

    public void setWattage(int wattage) {
        this.wattage = wattage;
    }

    @Override
    public String toString() {
        return super.toString() + " | " + wattage + "W";
    }
}
